package.json

```diff
@@ -15,5 +15,9 @@
   "devDependencies": {
     "husky": "^8.0.3",
     "turbo": "^1.9.6"
+  },
+  "dependencies": {
+    "@types/react-syntax-highlighter": "^15.5.7",
+    "react-syntax-highlighter": "^15.5.0"
```

Comment:
We should move these dependencies into `nextjs-website` module. Moreover the `@type` should be installed as `devDependecies`